pom.xml

```diff
@@ -1911,6 +1911,7 @@
                 <param>
                   com.puppycrawl.tools.checkstyle.checks.naming.PatternVariableNameCheck*
                 </param>
+                <param>com.puppycrawl.tools.checkstyle.checks.naming.RecordTypeParameterNameCheck*</param>
```

Comment:
Why is this naming check added to the pitest-misc group?

Reply:
Fixed.